src/code/components/animated-sprite.js

```diff
@@ -0,0 +1,42 @@
+import React, { PropTypes } from 'react';
+const tweenState = require('react-tween-state');
```

Comment:
You need to do a `yarn add react-tween-state` and commit the `package.json` and `yarn.lock` files as well.